starter-code/app.js

```diff
@@ -1,50 +1,113 @@
-const express      = require('express');
-const path         = require('path');
-const favicon      = require('serve-favicon');
-const logger       = require('morgan');
+'use strict';
+
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
 const cookieParser = require('cookie-parser');
-const bodyParser   = require('body-parser');
-const mongoose     = require("mongoose");
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const expressLayouts = require('express-ejs-layouts');
+const session = require('express-session');
+const MongoStore = require('connect-mongo')(session);
+const passport = require('passport');
+const FbStrategy = require('passport-facebook').Strategy;
 
-const app = express();
+const User = require('./models/user');
+
+const auth = require('./routes/auth');
 
 // Controllers
-const siteController = require("./routes/siteController");
+const siteController = require('./routes/siteController');
+
+const app = express();
 
 // Mongoose configuration
-mongoose.connect("mongodb://localhost/ibi-ironhack");
+mongoose.Promise = Promise;
+mongoose.connect('mongodb://localhost/ibi-ironhack', {
+  keepAlive: true,
+  reconnectTries: Number.MAX_VALUE
+});
+
+passport.use(new FbStrategy({
```

Comment:
Better put the passport configuration in its own file (inside a folder like helpers, for example).